lib/util/jsonRefs.ts

```diff
@@ -0,0 +1,10 @@
+import * as jsonRefs from "json-refs"
+import * as sm from "@ts-common/string-map"
+
+export { UnresolvedRefDetails } from "json-refs"
+
+export const findRefs = (
```

Comment:
do we really need this ?

Reply:
To make it strongly typed. `json-refs` has not-complete type definition https://github.com/whitlockjc/json-refs/pull/151

Reply:
ok so i suppose its a temporary workaround, can you refer the issue please so we can update once that is fixed